### ISSUE TITLE
Encoder will not allow upload past 110MB

### ISSUE THREAD

**Issue by KenRHill** [WWBN/AVideo-Encoder#50]
Hi there.  I have installed YouPHPTube v 5.01 and the private encoder on a dedicated Ubuntu 16.04 LTS 64-bit server.  When I upload an .MP4 file to the encoder from the streamer site that is less than 110MB and it uploads the file, and then it is converted and sent back to the streamer site like it should be.  If I try to upload a file any larger than 110MB the file instantly turns red after I select it and it won't upload at all.  I have modified my /etc/php/7.0/apache2/php.ini with the following variables.

memory_limit = 4096M
post_max_size = 2048M
upload_max_filesize = 2048M
max_execution_time = 7500
max_input_time = 7500

I have restarted apache2 after editing the /etc/php/7.0/apache2/php.ini file as well as rebooted the server.  I have cleared my browser cache.  I even tried to upload different larger than 110MB .MP4 files from a couple other devices and the problem still persists.  

If the file size is under 110MB the file uploads and converts and is sent back flawlessly, but if the file is over 110MB when you choose the file to upload to the encoder, the file name immediately turns red and it will not upload.

I am totally stumped on what could be causing the issue.  What could the problem be?  Any help is greatly apperciated.  I am willing to try anything to get it to work right.

Please see screenshots below or visit the site and try uploading a file larger than 110MB for yourself.

![redupload](https://user-images.githubusercontent.com/37643777/38856980-e13fb698-41dc-11e8-8e7b-6572acf2ddb2.jpg)
![2gbsize](https://user-images.githubusercontent.com/37643777/38856994-e84be5a6-41dc-11e8-965a-2da0c66c3609.jpg)

The Streamer Site is:
http://www.videorefuge.com/
The Encoder is:
http://www.videorefuge.com/encoder/

**Comment by DanielnetoDotCom** [comment]
Hi, please check you log files

**Comment by KenRHill** [comment]
I have two log files titled youphptube.log on the server.  If there are others, please let me know.

The first file resides in the /var/www/html directory, which for upload purposes here I renamed youphptube1.log

[youphptube1.log](https://github.com/DanielnetoDotCom/YouPHPTube-Encoder/files/1920581/youphptube1.log)

The second file resides in /var/www/html/encoder directory, which for upload purposes here I renamed youphptube2.log

[youphptube2.log](https://github.com/DanielnetoDotCom/YouPHPTube-Encoder/files/1920582/youphptube2.log)

**Comment by DanielnetoDotCom** [comment]
I am not sure why is it happening.
Please do this
1- clear all your log,
2- try to upload a file
3- paste your encoder log here
5- tell us the file name

**Comment by KenRHill** [comment]
I went into the /var/www/html/encoder directory. I deleted the youphptube.log.

Then, I went back to the streamer site, logged in and went to upload a file to the encoder.  The file name I tried is ScopeMount.mp4.  It is 1.45GB.  So...  From the encoder I choose Browse.  Picked the file and it immediately turns red and won't upload.  See screenshot.

![reduploadfile](https://user-images.githubusercontent.com/37643777/38907445-a34afa20-4270-11e8-887a-dd2df0b77b52.jpg)

I then went back into the /var/www/html/encoder directory.  There is no youphptube.log file.  Nothing was saved to the log file.

Then I went back to the streamer site to try a smaller file such as one that is less than 110MB.  File name this time is now TutorialEditionIntro.mp4.  File size is 25.82MB.  After selecting it, the upload starts immediately with no problem and encodes properly and is sent back to the streamer site.  See screenshot below.

![uploading](https://user-images.githubusercontent.com/37643777/38907658-98458c8e-4271-11e8-9d02-ad81cccab164.jpg)

I then went to check the /var/www/html/encoder directory to see if their was a log file and now I have a log file present.

[youphptube.log](https://github.com/DanielnetoDotCom/YouPHPTube-Encoder/files/1922220/youphptube.log)

It appears that any file larger than 110MB won't even start to upload at all.  Do you see anything wrong?  Possibilities even?

**Comment by KenRHill** [comment]
Update...  I found the cause of the problem.  I was using CloudFlare services to resolve DNS for my domain name to the dedicated server.  I had enabled DNS to CDN Proxy and it is limited to 100MB uploads on a free CloudFlare account and 500MB on a Enterprise account.  The solution for me, was to disable the DNS to CDN Proxy in CloudFlare.  This seems to have immediatly fixed the problem I was experiencing.  Hope this helps somebody else out there.
